app-games/src/main/java/com/aptoide/android/aptoidegames/feature_oos/OutOfSpaceDialog.kt

```diff
@@ -77,51 +76,35 @@ fun OutOfSpaceDialog(
         Message(requiredSpace)
         OutOfSpaceAppsList(packageName)
       }
-      GoBackButton(
-        backgroundColor = if (requiredSpace > 0) {
-          AppTheme.colors.outOfSpaceDialogGoBackButtonColor
-        } else {
-          AppTheme.colors.outOfSpaceDialogGoBackButtonEnoughSpaceColor
-        },
-        textColor = if (requiredSpace > 0) {
-          AppTheme.colors.outOfSpaceDialogGoBackButtonTextColor
-        } else {
-          AppTheme.colors.outOfSpaceDialogGoBackButtonEnoughSpaceTextColor
-        },
-        onDismiss = onDismiss,
-      )
+      if (requiredSpace > 0) {
```

Comment:
NOTE: Align with design/requirements to check if the button should be enabled or not in case there is still not enough space on the device

The only difference between the buttons is in the style. This condition can be applied inside the style in a single button composable.